src/python/grpcio/grpc/_server.py

```diff
@@ -817,3 +827,10 @@ def stop(self, grace):
 
     def __del__(self):
         _stop(self._state, None)
+
+
+def create_server(thread_pool, generic_rpc_handlers, interceptors, options,
```

Comment:
Suggestion: the arguments to this function are numerous and hard to remember. How about naming all of them?

Reply:
... for the moment I still like following the practice "always pass positional arguments positionally and keyword arguments by keyword".

But you do have a point. Maybe in the future there will be a role to play for a `ServerParameters` object or something?